### ISSUE TITLE
Declaration of AppKerne_::getCacheDir() must be compatible with Sylius\Bundle\CoreBundle\Application\Kernel::getCacheDir(): string

### ISSUE THREAD

**Issue by cdtweb** [Sylius/Sylius#8757]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Sylius version   | 1.0.0

Installing on PHP 7.2 results in return type mismatch (error below).

This issue exists in `vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php` and affects both `getLogDir()` and `getCacheDir()`.

```
Step 2 of 4. Setting up the database.
-------------------------------------

Creating Sylius database for environment dev.
 2/3 [░░░░░░░░░░░░░░░░░░░         ]  66%PHP Fatal error:  Declaration of AppKerne_::getCacheDir() must be compatible with Sylius\Bundle\CoreBundle\Application\Kernel::getCacheDir(): string in /Users/ctyler/home/sylius/var/cache/de_/kernel.tmp on line 49

Fatal error: Declaration of AppKerne_::getCacheDir() must be compatible with Sylius\Bundle\CoreBundle\Application\Kernel::getCacheDir(): string in /Users/ctyler/home/sylius/var/cache/de_/kernel.tmp on line 49


  [Symfony\Component\Debug\Exception\FatalErrorException]
  Compile Error: Declaration of AppKerne_::getCacheDir() must be compatible with Sylius\Bundle\CoreBundle\Application\Kernel::getCacheDir(): string
```

Without patching the Symfony bundle itself you could modify Sylius\Bundle\CoreBundle\Application\Kernel and remove the `string` return types on the affected methods.

**Comment by cdtweb** [comment]
I wanted to follow up and say that switching to PHP 7.1 and the error doesn't occur. So this appears to be limited to PHP 7.2 for the time being.

**Comment by pamil** [comment]
Fixed by #9075 and will be released as v1.0.7.
